input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json

```diff
@@ -0,0 +1,91 @@
+{
+  "resourceType" : "StructureDefinition",
+  "id" : "cqf-notDoneReason",
+  "text" : {
+    "status" : "extensions"
+  },
+  "url" : "http://hl7.org/fhir/StructureDefinition/cqf-notDoneReason",
+  "version" : "5.0.0",
+  "name" : "CQFNotDoneReason",
+  "title": "CQF Not Done Reason",
+  "status" : "active",
+  "experimental" : false,
+  "date" : "2024-08-20",
+  "publisher" : "HL7 International / FHIR Infrastructure",
+  "contact": [
+    {
+      "name" : "Clinical Quality Information WG",
+      "telecom": [
+        {
+          "system": "url",
+          "value" : "http://www.hl7.org/Special/committees/fiwg"
+        }
+      ]
+    }
+  ],
+  "description" : "Indicates the reason the event was not done.",
+  "jurisdiction" : [
+    {
+      "coding" : [
+        {
+          "system" : "urn:iso:std:iso:3166",
+          "code" : "US"
+        }
+      ]
+    }
+  ],
+  "fhirVersion" : "4.0.1",
+  "mapping" : [
+    {
+      "identity" : "rim",
+      "uri" : "http://hl7.org/v3",
+      "name" : "RIM Mapping"
+    }
+  ],
+  "kind" : "complex-type",
+  "abstract" : false,
+  "context" : [
+    {
+      "type" : "element",
+      "expression" : "Resource"
```

Comment:
For notDoneReason, what resources specifically do we expect this to be usable on? In QICore, this is only in the ObservationCancelled profile, so is Observation the only one? I'd say let's start there